compatibility/build-ts-libs.sh

```diff
@@ -12,8 +12,10 @@ eval "$(./dev-env/bin/dade-assist)"
 # before fetching it in another step.
 HEAD_TARGET_DIR=${1:-compatibility/head_sdk}
 
+VERSION=${DAML_SDK_RELEASE_VERSION:-0.0.0}
```

Comment:
Why are you changing this? This script shouldn’t ever be run with `DAML_SDK_RELEASE_VERSION` set to anything.

Reply:
Thanks! You're right, that change was not meant to go out.